src/ReinforcementLearningFarm/src/algorithms/tabular/td_learner.jl

```diff
@@ -0,0 +1,81 @@
+export TDLearner
+
+using LinearAlgebra: dot
+using Distributions: pdf
+import Base.push!
+
+using ReinforcementLearningCore: AbstractLearner, TabularApproximator
+using Flux
+
+"""
+    TDLearner(;approximator, γ=1.0, method, n=0)
+
+Use temporal-difference method to estimate state value or state-action value.
+
+# Fields
+- `approximator` is `<:TabularApproximator`.
+- `γ=1.0`, discount rate.
+- `method`: only `:SARS` (Q-learning) is supported for the time being.
+- `n=0`: the number of time steps used minus 1.
+"""
+Base.@kwdef mutable struct TDLearner{M,A} <: AbstractLearner where {A<:TabularApproximator,M<:Symbol}
```

Comment:
Should we alias TDLearner{:SARS} to SARSLearner ?

Reply:
Would this follow standard naming conventions? Or did we use to have a SARSLearner? My only concern is that the TD reference is then dropped / missing from the name

Reply:
SARS is a TD method so it kind of goes without saying. But we don't have to do this right now. This is easy to do in a later PR.